README.md

```diff
@@ -58,18 +58,13 @@ jobs:
           polaris_accessToken: ${{ secrets.POLARIS_ACCESS_TOKEN }}
           polaris_application_name: "testapp1"
           polaris_project_name: "testproj1"
-          polaris_assessment_types: "[\"SCA\", \"SAST\"]"
+          polaris_assessment_types: "SCA,SAST"
```

Comment:
Do we require this change now? As we have not yet released this change from synopsys-action side.